src/BackgroundTasks/IO.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ILIAS\BackgroundTasks;
+
+/**
+ * Interface IO
+ *
+ * @package ILIAS\BackgroundTasks
+ *
+ *          The IO as a defined format of data passed between two tasks. IO MUST be serialisable since it will bes stored in the database or
+ *          somewhere else
+ */
+interface IO extends \Serializable {
```

Comment:
Yeah, I don't really care about the name. Value sounds indeed better. @Amstutz I'm interested in your approach! :+1: